.travis.yml

```diff
@@ -7,26 +7,22 @@ notifications:
 addons:
   apt:
     sources:
-      - llvm-toolchain-trusty-5.0
       - ubuntu-toolchain-r-test
     packages:
-      - clang-5.0
       - g++-7
       - lcov
 compiler:
-  - gcc # todo: choose compiler
-env:
-  - MATRIX_EVAL="CC=clang-5.0 && CXX=clang++-5.0"
+  - gcc
 
 install:
   - git submodule update --init --recursive
 script:
   - mkdir build ; cd build ; cmake -DCMAKE_BUILD_TYPE=Debug ..
   - make
   - ./durian_test
-after_success:
-  - cd ${TRAVIS_BUILD_DIR}
-  - lcov --directory . --capture --output-file coverage.info # capture coverage info
-  - lcov --remove coverage.info '/usr/*' --output-file coverage.info # filter out system
-  - lcov --list coverage.info #debug info
-  - bash <(curl -s https://codecov.io/bash) || echo "Codecov did not collect coverage reports"
+after_success: |
+  cd ${TRAVIS_BUILD_DIR} &&
```

Comment:
why change from Travis script steps to one joined script?

also, if this is supposed to happen sequentially, isn't `;` more appropriate than `&&`?

Reply:
Yeah that's true. I was having issues where it would throw out the output from lcov and just run gcov again and I thought this might help, but turns out you just need to pass the `-X gcov` flag to codecov.io's bash script. I'll change it back.